examples/linear_model/plot_poisson_regression_non_normal_loss.py

```diff
@@ -77,7 +77,7 @@
       .format(df.loc[df["ClaimNb"] == 0, "Exposure"].sum() /
               df["Exposure"].sum()))
 
-fig, (ax0, ax1, ax2) = plt.subplots(ncols=3, figsize=(16, 4))
+_, (ax0, ax1, ax2) = plt.subplots(ncols=3, figsize=(16, 4))
```

Comment:
Even here, I'm not sure it's a positive change.

fig, ax = plt.subplots(...)

is more readable than,
```
_, ax = ..
```
particularly in an example even if fig is not sure.